### ISSUE TITLE
Viewport - error reported

### ISSUE THREAD

**Issue by ATrogolo** [GoogleChrome/lighthouse#4528]
Hi there, 
I run into an issue related to Accessibility section: 
my website, for viewport settings, uses: 
`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">`

LightHouse shows me an **error**: 
_[user-scalable="no"] is used in the <meta name="viewport"> element or the [maximum-scale] attribute is less than 5. Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page._ 

Setting `user-scalable=yes` seems to fix the problem. 
Drawback: now the user can zoom in&out the page, which is not the desidered behaviour. 

Could you fix this? 

Thanks, 
Andrea

**Comment by patrickhulce** [comment]
Making the page zoomable is the explicit goal of this audit, so it is in fact "the desired behavior" :)

From the [axe documentation](https://dequeuniversity.com/rules/axe/1.1/meta-viewport):

> The user-scalable=no parameter inside the content attribute of <meta name="viewport"> element disables zooming on a page. The maximum-scale parameter limits the amount the user can zoom. This is problematic for people with low vision who rely on screen magnifiers to properly see the contents of a web page.

**Comment by ATrogolo** [comment]
Ok, I understand this is related to accessibility advices, but it should be considered an error? 
Main websites like Facebook, Treebo, Booking don't allow to zoom the page...
If zoom is allowed all users can zoom the page, bringing to UX problems, isn't it?

**Comment by patrickhulce** [comment]
If you're taking the time to thoughtfully implement a custom handler for zooming your content, then yes you can safely ignore the error from this audit. Most of the time though this is not what is happening. Straight from the [HTML spec](https://w3c.github.io/html/document-metadata.html#element-attrdef-meta-content)

> Authors should not suppress or limit the ability of users to resize a document, as this causes accessibility and usability issues. There may be specific use cases where preventing users from zooming may be appropriate, such as map applications – where custom zoom functionality is handled via scripting. However, in general this practice should be avoided, and HTML conformance checking tools should display a warning if they encounter these values.

Since everyone seems aligned on this position, Lighthouse will continue to flag these cases.
